todo.css

```diff
@@ -0,0 +1,36 @@
+body {
+  background-color: #FAECA9
+}
+
+.topBar {
+  font-family: sans-serif;
+  font-size: 1em;
+  text-align: center;
+  color: white;
+  position: fixed;
+  top:-3%;
```

Comment:
nope I meant just line 11. you need line 10 to keep the header fixed at top, but i think you don't need top: -3%